plone/app/content/browser/contents/workflow.py

```diff
@@ -45,7 +44,7 @@ class WorkflowActionView(ContentsBaseAction):
 
     def __call__(self):
         self.pworkflow = getToolByName(self.context, "portal_workflow")
-        self.putils = getToolByName(self.context, "plone_utils")
+        self.plone_utils = getToolByName(self.context, "plone_utils")
```

Comment:
I will approve, but renaming attributes like this can theoretically break people's code.

Reply:
`plone_utils` should not be used in code consumed by `Products.CMFPlone`, like this package.
  
In fact we do not need this in here anymore, because I factored out the functionality already into `plone.base.utils.check_default_page_via_view`.  

I will change the code here to use it.

Reply:
ok, done.